lib/src/themes/feature_resolver.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:vector_tile_renderer/src/themes/selector.dart';
+
+import '../../vector_tile_renderer.dart';
+
+/// Resolver for resolving the [LayerFeature]s, that a [ThemeLayer] should
+/// render.
+abstract class ThemeLayerFeatureResolver {
```

Comment:
perhaps we could rename this to `LayerFeatureResolver` since it doesn't have much to do with themes any more.